week_2/bus_stops_3/bus_stops_3.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+#include <map>
+#include <set>
+#include <string>
+
+int main() {
+  int Q;
+  std::cin >> Q;
+  std::string stop;
+  int stops_count;
+  int k = 1;
```

Comment:
@Ypostolov which info will be stored in var k? For what purpose we need it?

Reply:
in this variable we store new numbers for buses routs, we assign new route numbers according to the value of this variable